src/components/LibraryDetailPage.tsx

```diff
@@ -40,15 +41,15 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
     this.renderStages = this.renderStages.bind(this);
     this.state = {
       libraryStage: this.props.library.stages.library_stage,
-      registryStage: this.props.library.stages.registry_stage
+      registryStage: this.props.library.stages.registry_stage,
     };
   }
 
   renderItems(category: {[key: string]: string}): JSX.Element {
 
     // Only create LibraryDetailItems for fields which actually have a value.
     let fields = Object.keys(category).filter(label => category[label]).map(label =>
-      <LibraryDetailItem key={label} label={label} value={category[label]} />
+      <LibraryDetailItem key={label} label={label} value={`${category[label]}`} />
```

Comment:
Why this update?

Reply:
Hmm sounds good. Except values being boolean or numbers doesn't seem right...

Reply:
I think it's fine for `online_registration` to be a boolean (especially since it's the only one).  I agree that numbers would be slightly weird.  But we can reevaluate that if/when it comes up in the future.